ee/fleetd-chrome/src/tables/screen_lock.ts

```diff
@@ -0,0 +1,20 @@
+import Table from "./Table";
+export default class TableScreenLock extends Table {
+  name = "screen_lock";
+  columns = ["delay"];
+
+  async generate() {
+    let delay;
+
+    try {
+      const autoLockDelay = await new Promise((resolve) =>
+        chrome.idle.getAutoLockDelay(resolve)
```

Comment:
Not sure I understand this.
getAutoLockDelay does not require a call back. Assuming I understand correctly and resolve is a function, any reason to pass it down?